rename_labels.json

```diff
@@ -0,0 +1,161 @@
+{
+    "id": "rename_labels",
+    "summary": "Rename dimension labels",
+    "description": "Renames the labels of a dimension in the data cube. Afterwards, the dimension can be referenced with the new name.",
+    "categories": [
+        "cubes"
+    ],
+    "parameter_order": [
+        "data",
+        "dimension",
+        "labels"
+    ],
+    "parameters": {
+        "data": {
+            "description": "The data cube.",
+            "schema": [
+                {
+                    "type": "object",
+                    "subtype": "raster-cube"
+                }
+            ],
+            "required": true
+        },
+        "dimension": {
+            "description": "The name of the dimension to rename the labels for.",
+            "schema": {
+                "type": "string"
+            },
+            "required": true
+        },
+        "labels": {
+            "description": "The new names for the labels.\n\nIf an **array** is passed, the dimension labels in the data cube are expected to be enumerated so that the dimension labels map to the array indices and get replaced. If the dimension labels are not enumerated, a `LabelsNotEnumerated` is thrown. The number of dimension labels must be equal to the number of array elements, otherwise the error `LabelMismatch` is thrown.\n\nIf an **object** is passed, the object defines a (potentially incomplete) mapping of the source dimension labels to the target dimension labels. If one of the source dimension labels doesn't exist, a `LabelNotAvailable` error is thrown. If a target dimension label exists in the data cube, a `LabelExists` error is thrown.",
```

Comment:
> dimension labels in the data cube are expected to be enumerated so that the dimension labels map to the array indices and get replaced. If the dimension labels are not enumerated, a `LabelsNotEnumerated` is thrown.

I'm not sure what "enumeration" means in this context.

> If a target dimension label exists

 ... label already exists ...

Reply:
enumerated means zero-based numbering (0,1,2,3,...).
Improved the text accordingly.